### PR TITLE
responders fmt

### PR DESCRIPTION
**Pull request by jmervine**
per https://trello.com/c/2J9qDq5X/258-post-mvp-update-responders-formatting

screen shot 2016-12-09 at 6 39 34 pm

### CODE REVIEW

File: spec/lib/chat_ops/commands/responders_command_spec.rb [hunk 1]
```diff
@@ -20,17 +20,34 @@
       incident.responders << user1
       incident.save
 
-      expect(process("responders")).to return_response_matching(/#{prevent_highlights(user1.name)}/)
-      expect(process("responders #{incident.incident_id}")).to return_response_matching(/#{prevent_highlights(user1.name)}/)
-      expect(process("who #{incident.incident_id}")).to return_response_matching(/#{prevent_highlights(user1.name)}/)
+      expected = [
+        "Incident responders for incident `#{incident.incident_id}`:",
+        "- #{user1.name} <#{prevent_highlights(user1.at_handle)}>"
```
Comment (joshuatobin) [inline spec/lib/chat_ops/commands/responders_command_spec.rb]: Do you still need the `prevent_highlights` here?

**Comment by joshuatobin** [comment]
@jmervine There is one failing spec but otherwise looks good

**Comment by jmervine** [comment]
@joshuatobin specs should be clean now.

**Comment by joshuatobin** [comment]
👍 Nice job on the specs too

**Pull request merged**